### PR TITLE
fix(plugin-meetings): added missing orgId for v2 get meeting info requests

### PR DESCRIPTION
**Pull request by emanuallan**
Got from: https://github.com/webex/webex-js-sdk/pull/2206, seperated it so it would be it's own trackable pr. 

Ticket responsible for this pr: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-274567

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/utilv2.js [hunk 1]
```diff
@@ -72,12 +73,17 @@ describe('plugin-meetings', () => {
 
     describe('#getRequestBody', () => {
       it('for _PERSONAL_ROOM_', () => {
+        const webex = new WebexCore({
+          credentials: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJyZWFsbSI6Im15LXJlYWxtIn0.U16gzUsaRW1VVikJA2VeXRHPX716tG1_B42oxzy1UMk'
```
Comment (lalli-flores) [inline packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/utilv2.js:77]: Where did these credentials come from? is this a real value?

Comment (emanuallan) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/utilv2.js:77]: I got it from another test `packages/node_modules/@webex/webex-core/test/unit/spec/credentials/credentials.js` line 275

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js [hunk 2]
```diff
@@ -192,7 +192,7 @@ MeetingInfoUtil.getDestinationType = async (from) => {
  * @returns {Object} returns an object with {resource, method}
```
Comment (lalli-flores) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:192]: Can you add a comment that `webex` is now part of the options?

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js [hunk 3]
```diff
@@ -201,7 +201,7 @@ MeetingInfoUtil.getRequestBody = (options) => {
       break;
     case _PERSONAL_ROOM_:
       body.userId = destination;
-      body.orgId = ''; // TODO: when to use the org iD (fetch others PMR)
+      body.orgId = webex.credentials.getOrgId(); // TODO: when to use the org iD (fetch others PMR)
```
Comment (lalli-flores) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:204]: What does this TODO mean?

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/utilv2.js (hunk 1 above)
Comment (lalli-flores) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/utilv2.js:77]: I see, seems like an encoded base64 string for a JWT token. can you add a comment for that?

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js (hunk 3 above)
Comment (emanuallan) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:204]: https://github.com/webex/webex-js-sdk/pull/2206#discussion_r744543716

@arun3528 May need your opinion on this.

**Review by arun3528 — APPROVED** [review]

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js (hunk 3 above)
Comment (emanuallan) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:205]: @arun3528 Need your opinion on what to do with this TODO Comment. bg info: https://github.com/webex/webex-js-sdk/pull/2206#discussion_r744543716

Comment (lalli-flores) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:205]: I confirmed with the API team that the org Id that we need to send should correspond to that of the user for which we are trying to fetch PMR information. This code won't work when requesting PMR information of external org users, we need a different solution

Comment (marcin-bazyl) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:205]: ok, yes, then this fix only works for checking the logged in user's PMR or PMRs of other users from the same org, so it's still better than nothing and fixes the most common use case for the web app, but yeah a further, better fix is required. How are you planning to proceed with this? deliver this fix and raise a new Jira for the "other org" case? or do a proper fix for all cases in this existing jira/pr?

**Comment by lalli-flores** [comment]
Replaced by https://github.com/webex/webex-js-sdk/pull/2237

**Pull request closed without merging**